### ISSUE TITLE
Add field that displays higher level taxonomy separate from taxon full name

### ISSUE THREAD

**Issue by tlammer** [specify/specify7#659]
Geneva requested to be able to display the higher level taxon separate from the taxon full name

From Geneva:
Another important information is the higher classification of the identification. Canthis be shown in a read-only calculated field? (minimum Phylum, Class, Order,Family)?

**Comment by maxxxxxdlp** [comment]
Can't you add a field like `parent.name` to the form definition?
Also, `parent.parent.name` may work too

**Comment by grantfitzsimmons** [comment]
![image](https://user-images.githubusercontent.com/37256050/176780224-de240fbb-9fa2-4a65-92f1-0b320b8d34a1.png)

db: `kufish`

**Comment by benritchie** [comment]
@grantfitzsimmons - can you clarify what is new here?  is it just that adding parent.parent.name was previously broken and now works, or is there something else?  
specifically (asking as this would unblock some issues we are having) is there a way to specify a particular rank - e.g. to get the family name?  (in a way that works whether you are at genus, or species, or variety level?

**Comment by maxxxxxdlp** [comment]
Nothing has changed here in 7.7.0.
We just realized that this use case is already covered by an existing feature.
You can show information about a parent of a parent by setting field name to something like `parent.parent.name`.

At present, this does not allow to enforce a specific tree rank, but we have a separate ticket for that - https://github.com/specify/specify7/issues/805
